app/views/provider/admin/account/authentication_providers/new.html.slim

```diff
@@ -18,21 +18,21 @@ div class="pf-c-card"
 
       = render partial: 'provider/admin/account/authentication_providers/form/form', locals: {f: f}
 
-      = f.input :site, as: :patternfly_input, required: true, label: 'Realm or Site', hint: 'Ex: https://rh-sso.example.com/auth/realms/demo'
+      = f.input :site, as: :patternfly_input, required: true, label: 'Realm or Site', hint: Formtastic::I18n.t('authentication_provider/keycloak.site', scope: :hints)
 
       = f.actions do
         = f.commit_button I18n.t('provider.admin.authentication_providers.new.submit_button_label')
 
+p id="keycloak_hint_text" style="display:none" = Formtastic::I18n.t('authentication_provider/keycloak.site', scope: :hints)
```

Comment:
These are just hidden fields that hold the values for the hints, so we can use them in JavaScript.